addons/point_of_sale/static/src/js/Chrome.js

```diff
@@ -201,6 +205,12 @@ odoo.define('point_of_sale.Chrome', function(require) {
         }
 
         setupBarcodeParser() {
+            if (!this.env.pos.company.nomenclature_id) {
+                const errorMessage = this.env._t("The barcode nomenclature setting is not configured. " +
+                    "Make sure to configure it on your Point of Sale configuration settings");
```

Comment:
Almost there. I know it doesn't look good, but AFAIK, only static string can be translated so it should be written as a single string.

```suggestion
                const errorMessage = this.env._t("The barcode nomenclature setting is not configured. Make sure to configure it on your Point of Sale configuration settings");
```

Reply:
are you sure ? I saw some cases on multiples lines like:
https://github.com/odoo/odoo/blob/16.0/addons/point_of_sale/static/src/js/Misc/AbstractReceiptScreen.js#L50-L52

Reply:
True. I'm wrong. Looks like it's allowed: https://www.odoo.com/documentation/16.0/developer/howtos/translations.html#blocks

Reply:
don't forget to include the pot file